docs/codacy-api/examples/uploading-dast-results.md

```diff
@@ -0,0 +1,77 @@
+---
+description: Instructions on how to upload DAST results to Codacy using the API.
+---
+
+# Uploading DAST results to Codacy
+
+To ensure the security of your web applications, Codacy allows you to upload DAST (dynamic application security testing) results from [Zed Attack Proxy (ZAP)](https://www.zaproxy.org/) directly to Codacy and monitor them as findings under [Security and risk management](../../organizations/managing-security-and-risk.md).
+
+## Uploading app scanning results to Codacy
```

Comment:
We also have an endpoint to retrieve uploaded reports to see their state. We should add that to the steps below.

https://api.codacy.com/api/api-docs#listdastreports

Reply:
Thanks. [Added](https://github.com/codacy/docs/pull/2150/commits/49f34558cac478b5a9fb63dc1ed8901972299499)